app/src/screens/MainScreen.js

```diff
@@ -20,10 +25,20 @@ import freeMoneyIcon from '../assets/freeMoneyIcon.svg'
 import howItWorksIcon from '../assets/howItWorksIcon.svg'
 
 const MainScreen = React.memo(({ isLoading }) => {
+  const { config } = useAppState()
   const theme = useTheme()
   const { below } = useViewport()
   const compact = below('medium')
 
+  const { currentPeriod } = useClock()
+
+  const { period, fetchingPeriodData } = usePeriod(currentPeriod)
+  const { totalRegisteredUsers, individualPayout } = period
```

Comment:
Let's fallback to default values when the period entity is not created. 

```suggestion
const { totalRegisteredUsers = bigNum(0), individualPayout = bigNum(0) } =
    period || {}
```